### PR TITLE
[FIX] partner_delivery_zone: Fix compute methods for all records

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

File: partner_delivery_zone/models/sale_order.py [hunk 1]
```diff
@@ -19,5 +19,4 @@ class SaleOrder(models.Model):
     @api.depends("partner_shipping_id")
     def _compute_delivery_zone_id(self):
         for so in self:
-            if so.partner_shipping_id.delivery_zone_id:
-                so.delivery_zone_id = so.partner_shipping_id.delivery_zone_id
+            so.delivery_zone_id = so.partner_shipping_id.delivery_zone_id
```
Comment (pedrobaeza) [inline partner_delivery_zone/models/sale_order.py]: That `if` was done on purpose if I don't remember bad, so you may need:
```suggestion
            if so.partner_shipping_id.delivery_zone_id:
                so.delivery_zone_id = so.partner_shipping_id.delivery_zone_id
            else:
                so.delivery_zone_id = so.delivery_zone_id  # reassign old value
```
(not sure this works)

**Comment by Tardo** [comment]
Check Travis

**Comment by victoralmau** [comment]
I think that everything it's OK

**Review by victoralmau — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
Not sure about not preserving old value if new delivery address has no delivery zone, but let's see.

**Pull request merged**